### ISSUE TITLE
Ice / water on land is transparent

### ISSUE THREAD

**Issue by GrimSqueaker** [GrimSqueaker/remc2#14]
Ice / water patches on land in the first level are see-through and the landscape behind it can be seen. It looks like a clipping error.

**Comment by GrimSqueaker** [comment]
I think these areas should reflect the sky but instead show the terrain behind the mountain.

**Comment by turican0** [comment]
Is this bug still up to date? I haven't discovered it yet, if so, please add a screenshot(or close this bug).
Thank you, Tom.

**Comment by GrimSqueaker** [comment]
Yes, it still is. In the marked area you can see the tree and windows of a building through the terrain.

![Screenshot from 2021-02-06 22-26-28](https://user-images.githubusercontent.com/26384847/107130107-ce5fcf00-68ca-11eb-8b1c-73a8e44473e3.png)

**Comment by turican0** [comment]
This is no regression, it is same in original game(when reflections turn on), Tom.
![image](https://user-images.githubusercontent.com/41465132/107148005-30631780-6951-11eb-8c0e-fd805a6c6c81.png)

**Comment by GrimSqueaker** [comment]
Ah. Then we need a new category of bug. To fix stuff that was even broken in the original.

**Comment by turican0** [comment]
Maybe the reflection is at the wrong angle.

**Comment by turican0** [comment]
May be I can change textures water of rivers, Tom.
